docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -33,57 +126,70 @@ Include it in your module file like this:
-* ``description:``
-
+:description:
   * A detailed description (generally two or more sentences).
   * Must be written in full sentences, i.e. with capital letters and fullstops.
   * Shouldn't mention the name module.
-* ``version_added:``
+:version_added:
   The version of Ansible when the module was added.
   This is a `string`, and not a float, i.e. ``version_added: "2.1"``
-* ``author:``
+:author:
   Name of the module author in the form ``First Last (@GitHubID)``. Use a multi-line list if there is more than one author.
-* ``options:``
-  One per module argument
+:options:
+  One per module argument:
+  
+  :option-name:
+
+    * Declarative operation (not CRUD)–this makes it easy for a user not to care what the existing state is, just about the final state, for example `online:`, rather than `is_online:`.
+    * The name of the option should be consistent with the rest of the module, as well as other modules in the same category.
 
-  * ``description:``
+  :description:
 
     * Detailed explanation of what this option does. It should be written in full sentences.
     * Should not list the options values (that's what ``choices:`` is for, though it should explain `what` the values do if they aren't obvious.
     * If an argument takes both True)/False and Yes)/No, the documentation should use True and False.
     * If an optional parameter is sometimes required this need to be reflected in the documentation, e.g. "Required when I(state=present)."
     * Mutually exclusive options must be documented as the final sentence on each of the options.
-  * ``required:``
+  :required:
     Only needed if true, otherwise it is assumed to be false.
-  * ``default:``
+  :default:
 
     * If `required` is false/missing, `default` may be specified (assumed 'null' if missing).
     * Ensure that the default parameter in the docs matches the default parameter in the code.
     * The default option must not be listed as part of the description.
-  * ``choices:``
+  :choices:
     List of option values. Should be absent if empty.
-  * ``aliases:``
+  :aliases:
     List of option name aliases; generally not needed.
-  * ``version_added:``
+  :version_added:
     Only needed if this option was extended after initial Ansible release, i.e. this is greater than the top level `version_added` field.
     This is a string, and not a float, i.e. ``version_added: "2.3"``.
-* ``requirements:``
+  :requirements:
     List of requirements, and minimum versions (if applicable)
-* ``notes:``
+  :notes:
     Details of any important information that doesn't fit in one of the above sections; for example if ``check_mode`` isn't supported, or a link to external documentation.
 
 
+.. note::
+
+   The above fields are are all in lowercase.
+
+.. note::
+
+   Their is no need to document the ``type:`` of an option.
+
+.. note::
 
+   If you module doesn't doesn't have any options (perhaps it's a ``_facts`` module)  you can use ``options: {}``.
```

Comment:
Change to "If your module doesn't doesn't have any options (for example, if it's a ``_facts`` module), you can use..."